tests/src/gui/testqgsrelationreferencewidget.cpp

```diff
@@ -572,24 +572,6 @@ void TestQgsRelationReferenceWidget::testIdentifyOnMap()
   w.featureIdentified( feature );
   QCOMPARE( w.mComboBox->currentData( Qt::DisplayRole ).toInt(), 10 );
 
```

Comment:
```suggestion
    w.setReadOnlySelector( true );
    QVERIFY( !w.mComboBox->enabled() );
```

Should we readd a minimal test like this?